README.md

```diff
@@ -6,136 +6,59 @@ MCP Server for Interacting with Salesforce Orgs
-       }
-     }
-   }
-   ```
-
-   You can also configure the MCP server globally by editing the VS Code [settings.json](https://code.visualstudio.com/docs/configure/settings#_settings-file-locations) file and adding a similar JSON snippet but contained in an `mcp:servers` section.
-
-   The `--orgs` flag is required and specifies the authorized orgs you're allowing the MCP server to access. The `--toolsets` and `--tools` flags are used to specify which tools the server will start with. See [Configure the DX MCP Server](README.md#configure-the-dx-mcp-server) for the available values for these flags.
-
-1. Open VS Code, go to **View -> Command Palette** and enter **MCP: List Servers**.
-
-   TIP: You can also get to the command palette by pressing press Ctrl+Shift+P (Windows or Linux) or Command-Shift-P (macOS).
-
-1. Click `Salesforce DX`, then **Start Server**.
-
-   Check the Output tab for the server status.
-
-1. Run **Chat: Open Chat (Agent)** from the command palette to start a new GitHub Copilot chat session.
-
-   Be sure your Copilot chat window is in `Agent` mode; if you're in `Ask` or `Edit` mode, use the [little drop-down](https://github.blog/ai-and-ml/github-copilot/copilot-ask-edit-and-agent-modes-what-they-do-and-when-to-use-them/) to switch.
+## Overview of the Salesforce DX MCP Server (Beta)
 
-1. In the GitHub Copilot chat window, use natural language to explain what you want to do. The MCP server determines which configured tool to use, and then shows it to you along with other information. Click **Continue** to run the tool and see the results of your request.
-
-   Try out these examples:
-
-   - List all my orgs.
-   - Which are my active scratch orgs?
-   - Show me all the accounts in the org with alias my-org.
-   - Deploy everything in my project to the org with alias my-org.
-   - Do you see any performance or security issues with the Apex code in the `MyApexClass.cls` file?
-   - I see that my Apex code violates the pmd:ApexCRUDViolation rule; can you give me more information about this rule?
+The Salesforce DX MCP Server is a specialized Model Context Protocol (MCP) implementation designed to facilitate seamless interaction between large language models (LLMs) and Salesforce orgs. This MCP server provides a robust set of tools and capabilities that enable LLMs to read, manage, and operate Salesforce resources securely.
 
-1. To stop, restart, or view the MCP server configuration, run the **MCP: List Servers** command, click `Salesforce DX`, then click the appropriate option.
+> **NOTE**: _Salesforce DX MCP Server is a pilot or beta service that is subject to the Beta Services Terms at [Agreements - Salesforce.com](https://www.salesforce.com/company/legal/) or a written Unified Pilot Agreement if executed by Customer, and applicable terms in the [Product Terms Directory](https://ptd.salesforce.com/). Use of this pilot or beta service is at the Customer's sole discretion._
```

Comment:
If you wanted to you could use Github special block quote syntax to make this stand out 
Details here https://github.com/orgs/community/discussions/16925